source/guide_gitea.rst

```diff
@@ -355,6 +309,198 @@ To update do:
 * Start the application ``supervisorctl start gitea``
 * Check if the application is running ``supervisorctl status gitea``
 
+You can also automate the update by using the ``gitea-update`` script.
+
+Create ``~/bin/gitea-update`` with the following content:
```

Comment:
there is still some `isabell` etc in here that needs to be replaced. I am unsure if this shouldn't be externalised in a specific repo. We have scripts like this in some guides and I find it useful and valuable, but we might need a better solution for it in the future. For now I think its ok to have it here and thanks for the approach.